K/Kira/build_tarballs.jl

```diff
@@ -0,0 +1,81 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+using Base.BinaryPlatforms
+const YGGDRASIL_DIR = (dirname∘dirname∘dirname)(@__FILE__)
+(include∘joinpath)(YGGDRASIL_DIR, "platforms", "mpi.jl")
+
+name = "Kira"
+version = v"2.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://gitlab.com/kira-pyred/kira.git", "8001d64f23406d6c4cb06578b89011c1a0c04c65"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+pip3 install -U meson
+
+mkdir ${WORKSPACE}/srcdir/Kira-build
+cd ${WORKSPACE}/srcdir/Kira-build/
+
+meson setup \
+    -Dfirefly=true \
+    -Djemalloc=true \
+    --cross-file=${MESON_TARGET_TOOLCHAIN} \
+    --buildtype=release \
+    ${WORKSPACE}/srcdir/kira/ \
+    ${WORKSPACE}/srcdir/Kira-build/
+sed -i "s/\/workspace\/destdir\/opt/\/opt/g" build.ninja
```

Comment:
Why do you need to do this?  Also, if you really need to do this
```suggestion
sed -i "s?/workspace/destdir/opt?/opt?g" build.ninja
```
is more readable

Reply:
See #6488.
I have checked the meson build configurations in this project.
However, I didn’t see the cause of the problem.
Therefore, I use this line for fixing it.